chef/data_bags/crowbar/migrate/crowbar/103_chef-solr-tmpfs.rb

```diff
@@ -0,0 +1,9 @@
+def upgrade(ta, td, a, d)
+  a["chef"]["solr_tmpfs"] = ta["chef"]["solr_tmpfs"]
```

Comment:
I understand you create one migration per commit, but it'd be fine to have only one migration for the pull request. In case you want to keep it that way, make sure to bump the revision in the json file (it's still 102)